mmdet3d/core/visualizer/image_vis.py

```diff
@@ -182,6 +182,8 @@ def draw_camera_bbox3d_on_img(bboxes3d,
 
     img = raw_img.copy()
     cam2img = copy.deepcopy(cam2img)
+    if len(bboxes3d.tensor) is 0:
```

Comment:
I think `if bboxes3d.tensor.shape[0] == 0` would be better? Also please add a comment indicating that `# in case there is no boxes`

Reply:
BTW just to make sure if this modification is necessary, the code will cause an error if no box is presented right?